exonum-light-client/src/main/java/com/exonum/client/HealthCheckResponse.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.client;
+
+import com.google.common.base.MoreObjects;
+
+class HealthCheckResponse {
+  boolean connectivity;
+
+  @Override
+  public String toString() {
```

Comment:
we can add `lombok` to the LC module and use this feature: https://projectlombok.org/features/Value
so this(and other DTOs) class will look like:
```
@Value
class HealthCheckResponse {
  boolean connectivity;
}
```